apps/examples/src/examples/speech-bubble/SpeechBubble/helpers.tsx

```diff
@@ -234,28 +181,13 @@ export function intersectLineSegmentLineSegment(
 /**
  * Inverse linear interpolation
  */
-export function invLerp(a: number, b: number, v: number) {
+function invLerp(a: number, b: number, v: number) {
 	return (v - a) / (b - a)
 }
 /**
  * Maps a value from one range to another.
  * e.g. mapRange(10, 20, 50, 100, 15) => 75
  */
-export function mapRange(a1: number, b1: number, a2: number, b2: number, s: number) {
+function mapRange(a1: number, b1: number, a2: number, b2: number, s: number) {
```

Comment:
If they're not used elsewhere, don't export them; it makes it hard to find dead code later on since exports are always treated as "used variables" by your IDE